packages/protoplugin-test/src/tagged-template-literal.test.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
```

Comment:
I would move these tests back into `transpile.test.ts`.  Not sure why, but it looks like the tests are split now between that file and this one.  I'd just create a nested `describe` block called `print with tagged template literal` inside the main `transpile` one.  

Also let's keep the `transpile` helper in the transpile test file since moving the tests back will keep everything contained in one place.

Reply:
@smaye81 
I split them because `transpile.test.ts` seems a bit long for me. But I'm not very picky about it, so fixed in [7c5fb57](https://github.com/bufbuild/protobuf-es/pull/279/commits/7c5fb57bbc131c5cdcd32faaa0f54424a47d3463).